.github/workflows/dispatch_release.yml

```diff
@@ -0,0 +1,17 @@
+name: Update homebrew
+
+on:
+  release:
+    types: [ published ]
```

Comment:
Does this fire for pre-releases too? Not sure this is a good idea. It feels hard to go back if a release turns out to be broken.

Reply:
Changed to "released". Hopefully that's what we want.